dashboard/src/components/AppList/AppList.tsx

```diff
@@ -94,16 +102,18 @@ function AppList() {
               value={searchFilter}
               submitFilters={submitSearchFilter}
             />
-            <CdsToggleGroup className="flex-v-center">
-              <CdsToggle>
-                <label>Show apps in all namespaces</label>
-                <input
-                  type="checkbox"
-                  onChange={toggleListAllNS}
-                  checked={allNSQuery === "yes" || allNS}
-                />
-              </CdsToggle>
-            </CdsToggleGroup>
+            {canSetAllNS && (
+              <CdsToggleGroup className="flex-v-center">
+                <CdsToggle>
+                  <label>Show apps in all namespaces</label>
+                  <input
+                    type="checkbox"
+                    onChange={toggleListAllNS}
+                    checked={allNSQuery === "yes" || allNS}
```

Comment:
Just out of curiosity: why did you choose "yes"/"no" instead of "true"/"false" ?

Reply:
it's closer to what a human would say? :smile: It was just to make clear that this is not automatically parsed as a boolean